scene/gui/control.cpp

```diff
@@ -1649,32 +1649,43 @@ Vector2 Control::get_combined_pivot_offset() const {
 
 /// Sizes.
 
-void Control::_update_minimum_size() {
+void Control::_update_size_bounds() {
 	if (!is_inside_tree()) {
-		data.updating_last_minimum_size = false;
+		data.updating_last_size_bounds = false;
 		return;
 	}
 
 	Size2 minsize = get_combined_minimum_size();
-	data.updating_last_minimum_size = false;
+	Size2 maxsize = get_combined_maximum_size();
+	data.updating_last_size_bounds = false;
+
+	if (minsize != data.last_minimum_size || maxsize != data.last_maximum_size) {
+		Size2 old_minsize = data.last_minimum_size;
+		if ((data.size_cache.is_equal_approx(old_minsize)) && (minsize.width < old_minsize.width || minsize.height < old_minsize.height)) {
```

Comment:
```suggestion
		if (data.size_cache.is_equal_approx(old_minsize) && (minsize.width < old_minsize.width || minsize.height < old_minsize.height)) {
```
Confusing parentheses